main.go

```diff
@@ -1,167 +1,97 @@
 package main
 
 import (
-	"encoding/json"
 	"errors"
 	"fmt"
-	"io/ioutil"
-	"log"
-	"os"
-	"path"
+	"github.com/sensu/sensu-go/types"
```

Comment:
`types` is actually an alias to our new package renaming. In sensu-go, we typically import the package as `corev2 "github.com/sensu/sensu-go/api/core/v2"`. In this case, all instances of `types.Event` would be replaced with `corev2.Event`.

Reply:
There is no way you could have known this without digging into the sensu-go code. But this would be a nice improvement (for consistency sake) if we implemented this in our plugins.